pages/linux/apx.md

```diff
@@ -0,0 +1,33 @@
+# apx
+
+> Package management utility for Vanilla OS.
+> It installs packages inside managed containers or directly inside the host.
+> More information: <https://github.com/Vanilla-OS/apx>.
+
+- Install a package in the system and also initialize the container:
+
+`sudo apx install --sys {{package}} && apx init`
+
+- Install package(s) in system:
+
+`sudo apx install --sys {{package1 package2 ...}}`
+
+- Install a package from AUR (Use `apx --aur run {{package}}` , `apx --aur remove {{package}}` to run or remove an installed package):
```

Comment:
IMO, we can leave the extra commands here instead of creating a separate page. What do you guys think?

Reply:
I suggest use something like `sudo apx {{--sys|--aur}} install {{package}}` to include sys and aur in one example.